src/ck-core/cklocation.C

```diff
@@ -1552,8 +1553,13 @@ CkMigratable::~CkMigratable() {
 	thisIndexMax.dimension=0;
 }
 
-void CkMigratable::CkAbort(const char *why) const {
-	::CkAbort("CkMigratable '%s' aborting: %s",_chareTable[thisChareType]->name, why);
+void CkMigratable::CkAbort(const char *format, ...) const {
+	char newmsg[256];
+	va_list args;
+	va_start(args, format);
+	vsnprintf(newmsg, sizeof(newmsg), format, args);
+	va_end(args);
+	::CkAbort("CkMigratable '%s' aborting: %s",_chareTable[thisChareType]->name, newmsg);
```

Comment:
Would it be preferable to make the `CkMigratable` name message a separate call to `CkError` as in `ArrayElement`?

Reply:
Yeah, that's a good idea. Changed in the most recent commit.